hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/ReplicationManager.java

```diff
@@ -1312,6 +1325,42 @@ public double getInflightReplicationLimitFactor() {
       return inflightReplicationLimitFactor;
     }
 
+    @Config(key = "inflight.off-peak.hour",
+            type = ConfigType.STRING,
+            defaultValue = "-1",
+            reconfigurable = true,
+            tags = { SCM },
+            description = "Set the time information of the replication off-peak period. " +
+                    "The default value -1 means turning off this parameter. The normal" +
+                    " start time is 0 to 23 hours, separated by commas. For example, '0,1,8,9,12'"
```

Comment:
So this '0,1,8,9,12' means 00:00-00:59, 01:00-01:59, 08:00-08:59, 09:00-09:59, 12:00-12:59, right?  Could you put this explanation into the description too?

Reply:
Thanks, i have updated the code